.travis.yml

```diff
@@ -29,15 +29,6 @@ jobs:
     - git config --global user.name "ladybugbot"
     - pip install -U .
     - npx semantic-release
-  - stage: docs
```

Comment:
I would keep the separated stage for doc otherwise when there is no new semantic release it will overwrite the docs.

Reply:
Ooooooooh right...good catch! I'll fix this just now!

Reply:
Resolved in 0f904c2